go-controller/pkg/ovn/master.go

```diff
@@ -61,6 +63,17 @@ const (
 // TODO: Verify that the cluster was not already called with a different global subnet
 //  If true, then either quit or perform a complete reconfiguration of the cluster (recreate switches/routers with new subnet values)
 func (oc *Controller) StartClusterMaster(masterNodeName string) error {
+	// The gateway router need to be connected to the distributed router via a per-node join switch.
+	// We need a subnet allocator that allocates subnet for this per-node join switch. Use the 100.64.0.0/16
+	// or fd98::/64 network range with host bits set to 3. The allocator will start allocating subnet that has upto 6
+	// host IPs)
+	var joinSubnet string
+	if config.IPv6Mode {
+		joinSubnet = "fd98::/64"
+	} else {
+		joinSubnet = "100.64.0.0/16"
```

Comment:
@trozet perhaps eventually, but they've been this way since the beginning of ovnkube. Hopefully nobody chooses a ClusterCIDR that overlaps with these, and in fact we should be checking that the clusterCIDR doesn't but I don't think we currently do that.